obci/interfaces/bci/budzik/experimental/budzik_prototype.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from obci.configs import settings
+from obci.interfaces.bci.analysis_master import AnalysisMaster
+from obci.interfaces.bci.budzik.experimental.dummy_classifier import DummyClassifier
+from obci.analysis.buffers.auto_ring_buffer import AutoRingBuffer
+
+
+class BudzikPrototype(AnalysisMaster):
```

Comment:
Jest tak jak piszecie — podejmowanie decyzji to sprawa implementacji add_result. W przykładowej implementacji tego nie ma, bo miała ona tylko wypisywać coś na ekran. Jeśli chcecie, mogę ją rozwinąć i dodać przykładową klasyfikację.